Interface/AddOns/oUF_Neav/tags.lua

```diff
@@ -21,8 +21,9 @@ oUF.Tags.Methods['druidmana'] = function(unit)
     end
 end
 
+oUF.Tags.Events['pvptimer'] = 'PLAYER_ENTERING_WORLD PLAYER_FLAGS_CHANGED'
 oUF.Tags.Methods['pvptimer'] = function(unit)
-    if (not IsPVPTimerRunning()) then
+    if (not IsPVPTimerRunning() or GetPVPTimer() == 301000 or GetPVPTimer() == 999) then
```

Comment:
GetPVPTimer will return 301000 when you aren't marked for PvP and it will sometimes only return 999 after loading while marked. If you have ever noticed it showing a 0 that is due to the 999 bug.